src/api/getAllData.tsx

```diff
@@ -0,0 +1,272 @@
+import { getTeamsForOrg } from "./getTeamsForOrg";
+import { getReposForTeam } from "./getReposForTeam";
+import { getVulnsFromRepo } from "./getVulnsFromRepo";
+
+import { Org, Team, Topic, Repository} from "../utils/types";
+import { formatVulnData } from "../utils/functions";
+
+export const getAllData = (graphql:any,  orgLogin: string) => {
+  return getAllRawData(graphql, orgLogin)
+};
+
+export async function getAllRawData(graphql:any, orgLogin:string): Promise<any> {
+  let seen = new Set<string>
+
+  let seenTopics =new Map<string, Topic >(); 
```

Comment:
You should format your code here. I suggest getting a formatter like Prettier. It'll make it consistent through out all of the files

Reply:
Name: Prettier - Code formatter
Id: esbenp.prettier-vscode
Description: Code formatter using prettier
Version: 9.10.4
Publisher: Prettier
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode